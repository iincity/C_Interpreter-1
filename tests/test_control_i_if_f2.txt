

int
a , b ;

int main()
{
a = 1  ;
b = 2 ; 
if (b > a) 
{
	if (a > b) {
		print a;
	}
}
	return;
}
