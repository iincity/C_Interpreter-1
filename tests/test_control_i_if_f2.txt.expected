Parse Error: return statement with no expression in a function with a return type
Got SEMICOLON
